tests/mesos_test.py

```diff
@@ -205,13 +205,15 @@ def test_handle_error(self):
         assert self.task.is_failed
         assert self.task.is_done
 
-    def test_handle_unknown_terminal_event(self):
+    def test_handle_terminal_event_offer_timeout(self):
         self.task.started()
         event = mock_task_event(
             task_id=self.task_id,
             platform_type=None,
             terminal=True,
             success=False,
+            raw='failed due to offer timeout',
+            message='stop',
```

Comment:
These two lines will cause the same "AttributeError: 'str' object has no attribute 'get' " error, because we call `get` on `raw` and `message` [here](https://github.com/Yelp/Tron/blob/offer-timeout-bug/tron/mesos.py#L210). However, the tests will still pass because we catch those exceptions [here](https://github.com/Yelp/Tron/blob/offer-timeout-bug/tron/mesos.py#L230). Maybe write a separate test for `log_event_info` so that check that the logs we want are indeed being logged?

Reply:
It doesn't cause the exception, because now we only call get when platform_type isn't None.

There's a test for logging exceptions [here](https://github.com/Yelp/Tron/blob/offer-timeout-bug/tests/mesos_test.py#L245), I'll update it to include log.warning.